### ISSUE TITLE
Be able to upvote current song from sticky player

### ISSUE THREAD

**Issue by kyleturman** [aaronmoodie/slime-public#9]
![screen shot 2015-07-01 at 1 57 37 pm](https://cloud.githubusercontent.com/assets/879896/8461241/2b6b7576-1ff9-11e5-9fa8-cdc057f30458.png)

**Comment by aaronmoodie** [comment]
Yeah, I have this somewhat implemented, just need to finish it off.
